build.gradle.kts

```diff
@@ -119,37 +111,28 @@ tasks {
   getByName("knitPrepare").dependsOn(getTasksByName("dokka", true))
 
   withType<DokkaMultiModuleTask>().configureEach {
-    outputDirectory.set(docFolder())
+    outputDirectory.set(file("docs"))
     moduleName.set("Arrow")
   }
 
-  register<Delete>("cleanDocs") {
-    val folder = docFolder()
-    val content = folder.listFiles()?.filter { it != folder }
-    delete(content)
-  }
-
   register<Copy>("copyCNameFile") {
     from(layout.projectDirectory.dir("static").file("CNAME"))
     into(layout.projectDirectory.dir("docs"))
   }
 }
 
-fun docFolder(): File =
-  project.properties["githubpages"]?.let { file("docs").also { it.mkdir() } } ?: rootDir.resolve("arrow-site/docs/apidocs")
-
 apiValidation {
   val ignoreApiValidation = if (!enableCompatibilityMetadataVariant) {
-    listOf("arrow-optics-ksp-plugin", "arrow-site")
+    listOf("arrow-optics-ksp-plugin")
   } else {
     listOf("arrow-optics-ksp-plugin")
   }
 
```

Comment:
**suggestion:** We could remove `if/else` here, since both if and else are returning the same.

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org/) standard</sub>

Reply:
Whoops 😅 Yes, thanks for catching 😁